app/controllers/api/v3/providers_controller.rb

```diff
@@ -25,6 +31,22 @@ def show
                                        train_with_disability sites
                                        accredited_bodies accredited_body?] }
       end
+
+    private
+
+      def build_fields_for_index
+        @fields = default_fields
+
+        return if params[:fields].blank?
```

Comment:
Can we combine the guard below with this one here to reduce the cyclomatic complexity pls?